src/Tools/dotnet-monitor/CollectionRules/Triggers/ICollectionRuleTriggerOperations.cs

```diff
@@ -33,7 +33,7 @@ internal interface ICollectionRuleTriggerOperations
         /// </summary>
         bool TryValidateOptions(
             string triggerName,
-            object options,
+            object? options,
```

Comment:
We migrated most options here to be record types, which could guarantee non-nullability. If we push that approach across the board, we could potentially change this to non-nullable. I don't think there's a base type for records though.

Reply:
Updated https://github.com/dotnet/dotnet-monitor/issues/6929 to track this since it's out of scope of this PR.